### PR TITLE
Caa UI update

### PR DESCRIPTION
**Pull request by SharpRake**
## Type of change

Update to the cloud account association guide. This adds a section on setting up a CAA for GCP through the console. AWS is coming soon but that will be a much smaller addition.

### How should this PR be tested?

Test the new setup section (terraform sections still work as written and don't need testing). Also review changes to the overview section and the new "removing associations" section.

### CODE REVIEW

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/troubleshooting-tips.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/troubleshooting-tips.md:1]: can you remove this file from the PR?

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/cloud-account-associations/index.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+![Screenshot showing an example "Settings" tab, including a table with four groups. No group has yet been associated with a cloud account.](settings-grp-selection.png)
+
+To associate a group with a cloud account, find the group you want to associate and click on the pencil-shaped edit icon for the cloud of your choice. A modal window will appear from the right with next steps, which differ depending on the cloud provider you selected.
+
+> _Note: As of this writing, this feature is only available _in preview_. Additionally, setting up AWS account associations through the Chainguard Enforce Console is not available at this time. 
+
+
+### Setting up a GCP Cloud Account Association through the Console
+
+If you clicked the edit icon for GCP, the modal window will outline a four step process to set up a cloud account association.
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/cloud-account-associations/index.md:197]: Adjective

```suggestion
If you clicked the edit icon for GCP, the modal window will outline a four-step process to set up a cloud account association.

```

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/cloud-account-associations/index.md [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+
+In the second step, a dropdown menu will appear with all the projects available for you to choose. Select the project you want to associate with your Chainguard Enforce group and click **Continue**. 
+
+![Screenshot showing Step 2 of the GCP cloud account association process. There is a dropdown menu labeled "Project" with no items selected yet.](gcp_step2.png)
+
+#### Configure Workload Identity Federation
+
+After selecting a project, the next step is to configure workload identity federation. Rather than supplying a service account key — which would provide Chainguard Enforce with unnecessarily broad permissions — workload identity federation allows Enforce to impersonate service accounts living in your Cloud environment.
+
+If this is your first time setting up a cloud account association for this project you will need to explicitly authorize the creation of three things: the IAM Service Account Credentials API, the Workload Identity, and the Workload Identity Provider.
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/cloud-account-associations/index.md:214]:
```suggestion
If this is your first time setting up a cloud account association for this project, you will need to explicitly authorize the creation of three resources: 
* IAM Service Account Credentials API
* Workload Identity
* Workload Identity Provider

```

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/cloud-account-associations/index.md [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+Lastly, you need to configure a set of service accounts which will connect with Chainguard, allowing Enforce to perform certain limited functions on your project's resources.
+
+Again, if you haven't already set up the required service accounts on your chosen project then you must click the **Configure** button to do so.
+
+![Screenshot showing Step 4 of the GCP cloud account association process. This picture shows all six service account profiles are in a "Not Ready" state. There is a large "Incomplete" warning along with a "Configure" button above the table.](gcp_step4.png)
+
+Once each of the service accounts are in a **Ready** state, click the **Done** button to finish setting up your cloud account association.
+
+
+## Removing Cloud Account Associations
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/cloud-account-associations/index.md:231]: I tried to remove the association via Console, but that does not seem to be possible right now?

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
I made a few suggestions here. Can you also remove the troubleshooting tips file because that will be difficult to track if you make changes on another PR.

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/troubleshooting-tips.md (hunk 1 above)
Comment (SharpRake) [reply content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/troubleshooting-tips.md:1]: yikes! yes. will try to be more careful with this moving forward

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/cloud-account-associations/index.md (hunk 4 above)
Comment (SharpRake) [reply content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/cloud-account-associations/index.md:231]: Not now, and I'm not sure whether that's coming in the future. I'll add a note to clarify that it's not currently possible through the UI

**Review by ltagliaferri — APPROVED** [review]

**Pull request merged**